.evergreen/config.yml

```diff
@@ -228,6 +257,7 @@ buildvariants:
   - build-and-test-shared-bson
   - build-and-test-asan
   - build-and-test-java
+- name: suse12-s390x  
```

Comment:
Was this a bug?

Reply:
Yeah - appears to be two lots of settings but only one name